images/win/Windows2019-Readme.md

```diff
@@ -29,7 +29,7 @@
 - NPM 6.14.6
 - Yarn 1.22.4
 - pip 20.2.2 (python 3.7)
-- Miniconda 4.6.14
+- Miniconda 4.8.3

```

Comment:
documentation files (*.md) are auto-generated and updated automatically during deployment. Please revert them

Reply:
Ok, will do.